kairon/shared/concurrency/actors/utils.py

```
@@ -184,4 +187,22 @@ def send_waba_message(payload: dict, key: Text, bot: str, predefined_objects: di
+                bot = bot,
+                upload_status = UserMediaUploadStatus.completed.value,
+                media_id__ne = "",
+                upload_type__in = [UserMediaUploadType.user_uploaded.value, UserMediaUploadType.system_uploaded.value]
+            ).only("filename", "media_id")
+
+            if not media_data:
+                return []
+
+            return [{"filename": doc.filename, "media_id": doc.media_id} for doc in media_data]
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)